library/include/borealis/core/application.hpp

```diff
@@ -203,6 +213,11 @@ class Application
 
     inline static std::string commonFooter = "";
 
+    inline static bool globalQuitEnabled = false;
```

Comment:
Can you run clang-format please? `./scripts/format.sh --fix`

Reply:
I had to modify the script on my computer to run as I have clang 11 installed on my computer, and didn't see a way to easily install clang 8 on Arch. I did run it though. (I only committed the files that I've touched in this PR.) :+1: